src/components/LogbookContext/index.tsx

```diff
@@ -77,10 +79,11 @@ type ReducerAction =
     }
 
 type Context = {
+  getRecentLogbooks: (limit?: number) => Promise<void>
   getLogbook: (tokenId: string) => Promise<void>
   logbooks: { [tokenId: string]: Logbook | undefined }
 
-  getOwnNFTs: () => Promise<void>
+  getOwnNFTs: (owner?: string | null | undefined) => Promise<void>
```

Comment:
is @robertu7 aware of an address type?  to combine all `string | null | undefined` types?